### PR TITLE
Pog, Toast, Tooltip, Upsell, Video: add visual diff testing and screenshot to description in VSCode (Flow extension)

### PR DESCRIPTION
**Pull request by AlbertCarreras**
#### What changed?

Pog, Toast, Tooltip, Upsell, Video: add visual diff testing and screenshot to description in VSCode (Flow extension)

#### Why?

Provide basic visual testing coverage for most Gestalt components and link screenshots to VSCode's Flow extension that shows component screenshot below the component description on hover for better in-editor documentation.

OKR: 
- Tooling & Infra
- Visual Diff testing 
- Every component in our library uses visual diff testing to guard against UI regression

### CODE REVIEW

File: cypress/integration/visual-test/Pog-dark_spec.js [hunk 1]
```diff
@@ -0,0 +1,15 @@
+describe('Pog visual dark mode regression check', () => {
```
Comment (dangerismycat) [inline cypress/integration/visual-test/Pog-dark_spec.js:1]: Should these files be Flow typed? Have we been add Flow annotations to the other visual test files?

File: packages/gestalt/src/Pog.js [hunk 2]
```diff
@@ -89,7 +89,11 @@ type Props = {|
 /**
  * [Pog](https://gestalt.pinterest.systems/pog) is a lower-level functional component to show the active, hovered, & focused states of [IconButton](https://gestalt.pinterest.systems/pog/iconbutton).
  *
- *This abstraction to allow for links that look like IconButton.
+ * This abstraction to allow for links that look like IconButton.
```
Comment (dangerismycat) [inline packages/gestalt/src/Pog.js:92]: (pre-existing) typo: "This _is an_ abstraction to allow for…"

File: packages/gestalt/src/Toast.js [hunk 3]
```diff
@@ -40,6 +40,10 @@ type Props = {|
  * [Toasts](https://gestalt.pinterest.systems/toast) can educate people on the content of the screen, provide confirmation when people complete an action, or simply communicate a short message.
  *
  * Toast is purely visual. In order to properly handle the showing and dismissing of Toasts, as well as any animations, you will need to implement a Toast manager.
+ *
+ * ⚠️ Please note: Toast is not currently supported in dark mode.
```
Comment (dangerismycat) [inline packages/gestalt/src/Toast.js:44]: 🙏

File: packages/gestalt/src/Upsell.js [hunk 4]
```diff
@@ -131,7 +131,9 @@ type Props = {|
 /**
  * [Upsells](https://gestalt.pinterest.systems/upsell) are banners that display short messages that focus on promoting an action or upgrading something the user already has.
  *
- * ⚠️ Please note: Upsell is not currently supported in dark mode.
+ * ⚠️ Please note: Upsells is not currently supported in dark mode.
```
Comment (dangerismycat) [inline packages/gestalt/src/Upsell.js:134]: typo: Upsell should be singular

**Review by dangerismycat — APPROVED** [review]

File: cypress/integration/visual-test/Pog-dark_spec.js (hunk 1 above)
Comment (AlbertCarreras) [reply cypress/integration/visual-test/Pog-dark_spec.js:1]: Have we been add Flow annotations to the other visual test files?
NO

File: packages/gestalt/src/Pog.js (hunk 2 above)
Comment (AlbertCarreras) [reply packages/gestalt/src/Pog.js:92]: DONE

**Pull request merged**